client/layout/sidebar/item.jsx

```diff
@@ -37,7 +38,10 @@ export default React.createClass( {
 
 	render() {
 		const isExternalLink = isExternal( this.props.link );
-		const classes = classnames( this.props.className, { selected: this.props.selected } );
+		const classes = classnames( 'sidebar__item', this.props.className, {
```

Comment:
Agreed! Fixed in https://github.com/Automattic/wp-calypso/pull/14022/commits/ef650dfd7aae2ba7ec9df10dc3bcc3f09e067143